src/SFA.DAS.EAS.Portal.UnitTests/Portal/Database/Models/AccountTests.cs

```diff
@@ -19,27 +20,43 @@ public void ReservedFundingConstructor_WhenConstructWithReservedFunding_ThenShou
 
     public class AccountTestsFixture
     {
-        public Account Account { get; set; }
+        public AccountDocument AccountDocument { get; set; }
+        public Account Account{ get; set; }
```

Comment:
`Account`, `MessageId` and `CreatedDate` are not used and can be removed.

Reply:
Done.